src/components/TokenDetails.tsx

```diff
@@ -86,7 +91,8 @@ const TokenDetails = ({ id }: Props) => {
                   padding: [0, 0, "default"]
                 }}
                 onClick={() => {
-                  window.open(`${GENERATOR_URL}/${CORE_CONTRACT_ADDRESS?.toLowerCase()}/${token.tokenId}`)
+                  const generatorUrl = contractConfig[0].GENERATOR_URL
```

Comment:
is this supposed to be hardcoded as 0?

Reply:
I suppose it doesn't matter since the links are the same, but for robustness' sake should probably make this programatic depending on the token (in a world where someone might have 2 diff generator urls). A util function of some sort maybe?

Alternatively if generator link will ALWAYS be the same for everyone, maybe it is just its own config var instead of being in that contractConfig struct

Reply:
The 0 hardcoding was intentional (although probably bad practice) - I was using CONTRACT_INFO.filter() to get the proper contract config, and this returned an array for which there should have been only a single element. I have addressed this with [071be03](https://github.com/ArtBlocks/artblocks-engine-react/pull/10/commits/071be0363e6a4237b3733c943f00446b578e7f0c). Now using CONTRACT_INFO.find() to avoid the hardcoding.